### ISSUE TITLE
Weiss et al. (2015) summary displaying the Method codes incorrectly

### ISSUE THREAD

**Issue by njarboe** [earthref/MagIC#198]
Weiss et al. (2015) summary displaying the Method codes incorrectly.
![screen shot 2017-07-31 at 12 34 01 pm](https://user-images.githubusercontent.com/1278789/28794831-921d6e8c-75ec-11e7-9ece-c7bdad7b2951.png)

**Comment by rminnett** [comment]
It looks like there's an extra tab in most of the measurements and then the last column is missing so it didn't trip the upload parser with the wrong number of columns per row (full validation will catch this, though):

![image](https://user-images.githubusercontent.com/1182698/28795248-0498d572-75ee-11e7-838b-2c4dcdaa9390.png)

**Comment by njarboe** [comment]
On inspection it looks like just the column header software_package is missing. The number of tabs is correct. I changed "" to "software_package".  Once uploading is working I'll test it in a private space and then I think we should update the file for Caue with a new version number. I can then email a message what the problem was and tell him we fixed it and he can check out the new data file if he likes. If it is not too hard for me to upload on the back side it would be nice to have you show me what to do so I can do it in the future. Below is the fixed file.
[magic_contribution_11881Fixed.txt](https://github.com/earthref/MagIC/files/1307194/magic_contribution_11881Fixed.txt)

**Comment by rminnett** [comment]
I think the fixed file still has an extra tab starting on line 3569. Can you double check?

**Comment by njarboe** [comment]
Ok. Looks like there are a bunch of bad measurements where the samples aren't demaged or something. Lines 3569 to 14295. I've deleted them.  Sorry I didn't catch that on the image above. The font a bit small for me to see what was going on might have been the reason. Should have click on it to see it bigger. I did not see any other blocks of bad measurements in the rest of the file so hopefully this one is good. I guess I will wait to see it looking correct on the website before closing.
[magic_contribution_11881FixedLinesDeleted.txt](https://github.com/earthref/MagIC/files/1307436/magic_contribution_11881FixedLinesDeleted.txt)

**Comment by rminnett** [comment]
Great, thanks! Hopefully, we can get those bad measurements fixed and back in from Caue, too.

**Comment by njarboe** [comment]
Yeah. I'll tell him about it. Not sure if they were failed measurements or something that went wrong with the data processing.
